src/fastapi_cli/cli.py

```diff
@@ -34,7 +32,17 @@
         app as fastapi_cloud_cli,
     )
 
-    app.add_typer(fastapi_cloud_cli)
+    app.add_typer(fastapi_cloud_cli) 
+except ImportError:  # pragma: no cover
+    pass
+
+
+try:
+    from fastapi_new.cli import (  # type: ignore[import-not-found]
+        app as fastapi_new_cli,
+    )
+
+    app.add_typer(fastapi_new_cli) # pragma: no cover
```

Comment:
So, this is required because we don't install/test `new` as an optional dep in this repo, unlike cloud CLI which is included in `standard`.